sample-apps/react-native/dogfood/android/app/build.gradle

```diff
@@ -78,6 +78,23 @@ def reactNativeArchitectures() {
     return value ? value.split(",") : ["armeabi-v7a", "x86", "x86_64", "arm64-v8a"]
 }
 
+// Utility function to get the keychain password for MacOS
+def getPassword(String currentUser, String keyChain) {
+   def stdout = new ByteArrayOutputStream()
+   def stderr = new ByteArrayOutputStream()
+   exec {
+       commandLine 'security', '-q', 'find-generic-password', '-a', currentUser, '-s', keyChain, '-w'
+       standardOutput = stdout
+       errorOutput = stderr
+       ignoreExitValue true
+   }
+   //noinspection GroovyAssignabilityCheck
+      stdout.toString().trim()
+}
+
+// Call the `getPassword` function to get the release password
+def releasePassword = getPassword("Khushal Agarwal", "stream-video-calls-rn.keystore")
```

Comment:
i hope there are better ways than this..

We could store the file in the CI and then use that in firebase - I think

Reply:
and this seems  to be mac os sepcific, while we use ubuntu in CI